src/elm/Data/Expirable.elm

```diff
@@ -15,13 +16,17 @@ type SecondsRemaining
     = SecondsRemaining Time.Time
 
 
+type SecondsTotal
+    = SecondsTotal Time.Time
+
+
 type Expirable a
-    = Expirable a SecondsRemaining
+    = Expirable a SecondsRemaining SecondsTotal
 
 
 expiresIn : SecondsRemaining -> a -> Expirable a
```

Comment:
What do you think about `buildExpirableFromTotalSeconds`?

Reply:
Hmm - I'm not sure I'm onboard with that amount of verbosity, but I'm interested in hearing why you feel that better clarifies intent over `expiresIn`!

Reply:
I was looking for a name that expresses that this function builds something.  When first reading it, i expected the behavior to be such that it takes an Expirable and returns how many seconds are remaining.

Reply:
Ah, huzzah - how about `build`, which is a pattern used throughout the codebase? This can happen in a separate PR.